features/nanostack/mbed-mesh-api/source/WisunBorderRouter.cpp

```diff
@@ -186,3 +225,37 @@ int WisunBorderRouter::routing_table_get(ws_br_route_info_t *table_ptr, uint16_t
 
     return ws_bbr_routing_table_get(_mesh_if_id, (bbr_route_info_t *)table_ptr, table_len);
 }
+
+mesh_error_t WisunBorderRouter::set_radius_server_ipv6_address(const char *address)
```

Comment:
Do we need the GET interfaces for these

Could this be combined in the higher level api to have radius_configure(address,password)?

Reply:
Yes, I can add the get interface. There are no defaults for these, so the application must always program these first, but it might be easier for application, if the values could also be read from Nanostack.

I would prefer having the two functions for this since either one can be changed individually.

Reply:
Added get interfaces. Added storing of IP address and shared secret to border router class. Changed configuration to two phases, on class init the .json configuration is applied (if set) and then on BBR start the BBR is configured with the address and shared secret using the then define (mesh) interface id.